docs/libraries/http/reference/data-types.md

```diff
@@ -97,18 +97,18 @@ model TypeSpec.Http.BearerAuth
 
 Defines a model with a single property of the given type, marked with `@body`.
 
-This can be useful in situations where you cannot use a bare T as the body
+This can be useful in situations where you cannot use a bare type as the body
 and it is awkward to add a property.
 
 ```typespec
-model TypeSpec.Http.Body<T>
+model TypeSpec.Http.Body<TBody>
 ```
 
 #### Template Parameters
 
-| Name | Description                              |
-| ---- | ---------------------------------------- |
-| T    | The type of the model's `body` property. |
+| Name  | Description                              |
+| ----- | ---------------------------------------- |
+| TBody | The type of the model's `body` property. |
```

Comment:
It looks in other cases you dropped the T (TLocation -> Location) so why is this TBody?